src/solvers/td3.ml

```diff
@@ -287,7 +287,7 @@ module WP =
           let add_stmts (f: fundec) =
             List.iter (fun s -> Hashtbl.replace nodes (Node.show_id (Statement s)) ()) (f.sallstmts)
           in
-          List.iter (fun f -> Hashtbl.replace nodes (Node.show_id (FunctionEntry f)) (); Hashtbl.replace nodes (Node.show_id (Function f)) (); add_stmts f) functions;
+          List.iter (fun f -> Hashtbl.replace nodes (Node.show_id (FunctionEntry f)) (); Hashtbl.replace nodes (Node.show_id (Function f)) (); add_stmts f; Hashtbl.replace nodes (string_of_int (f.svar.vid + 10_000_000_000)) ()) functions;
```

Comment:
What is this magic number? If it's supposed to match some other places, it should probably be a constant variable, so it's not possible to accidentally change it in one place but forget about the others.

Reply:
This change is for also removing the outdated unknowns of a pseudo return because a pseudo return node from a function has the same id across different program versions. I tried to improve it by extracting the calculation of the pseudo return ids in efbe9e5ef0acc359cc5a03c9e5a0cb615970007a.